libs/transfer/src/lib/views/make-transfer-summary-view/make-transfer-summary-view.component.ts

```diff
@@ -7,13 +7,17 @@ import { MakeTransferJourneyState } from '../../services/make-transfer-journey-s
   templateUrl: 'make-transfer-summary-view.component.html',
 })
 export class MakeTransferSummaryViewComponent {
-  title = this.route.snapshot.data.title;
+  title = this.route.snapshot.data['title'];
```

Comment:
Dot notation, please.

Reply:
I changed because property 'title' comes from an index signature, so it must be accessed with ['title'].ts(4111)

I can disable noPropertyAccessFromIndexSignature if needed.

Reply:
It's ok.